src/data/endpoints.ts

```diff
@@ -167,6 +168,20 @@ export const endpointsMap: EndpointsMap = {
       },
     },
   },
+  fee_stats: {
+    label: "Fee Stats",
+    endpoints: {
+      all: {
+        label: "Fee Stats",
+        helpUrl: "https://developers.stellar.org/api/aggregations/fee-stats/object/",
+        method: RequestMethod.GET,
+        path: {
+            template: "/fee_stats",
+        },
+        setupComponent: Fees,
```

Comment:
If we show the message that there are no options, it will need to be styled.

![image](https://user-images.githubusercontent.com/7346473/150541673-fe4f0a8f-c0ec-42b7-8945-3c549724eb05.png)

We could leave it blank, though. Do you think it will be confusing to the users that there is nothing more to select? 

To render nothing, we could replace `Fees` with `() => null`.

![image](https://user-images.githubusercontent.com/7346473/150541708-70456ff8-f57d-4eb3-87e5-082f675eecae.png)

Reply:
I think rendering nothing looks better.  I tried a styled note but it seems superfluous.  I will change it to render nothing.

Reply:
![image](https://user-images.githubusercontent.com/35656692/150631694-493b687b-bca6-4b0a-ac20-8f33892bcea5.png)
This definitely looks better.